### PR TITLE
20 - Adding routes

### PR DESCRIPTION
**Pull request by aquintiliano**
…eated

### CODE REVIEW

**Comment by ghost** [comment]
Waiting for review from @lauroaraujo 

Also, please fix conflicts.

**Comment by aquintiliano** [comment]
conflicts fixed

File: Frontend/src/app/app.component.html
Comment (lauroaraujo) [inline Frontend/src/app/app.component.html]: Why is the router outlet inside a header (h1)? Seems odd...

File: Frontend/src/app/page-not-found/page-not-found.component.html
Comment (lauroaraujo) [inline Frontend/src/app/page-not-found/page-not-found.component.html:1]: The h1 tag from the app component should probably be here

**Review by lauroaraujo — CHANGES_REQUESTED** [review]
Looks fine, only one minor comment

**Comment by aquintiliano** [comment]
merging with Lauro authorization

**Pull request merged**
